views/forms/ControlledCheckbox/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { Controller } from 'react-hook-form'
+import { Checkbox } from '@material-ui/core'
+
+const ControlledCheckbox = (props) => {
+  return (
+    <Controller
+      name={props.name}
+      control={props.control}
+      render={({ field }) => (
+        <Checkbox
+          checked={field.value}
+          onChange={(e) => {
+            field.onChange(e)
+            props.onChange()
```

Comment:
why do we need both of these change handlers?

Reply:
do we need a custom onChange handler?

Reply:
To trigger the update event during an onChange, otherwise we update the form but no event is triggered

Reply:
Mike's out on parental leave, but for posterity the place where this is used is right here: https://github.com/AMP-SCZ/dpdash/pull/359/files#diff-3675c69c4ce5ee44bbfdecf612fbf83e9e2058d59dde578d81ae5c3935043e62R106